public/index.php

```diff
@@ -62,9 +70,10 @@
                     </div>
                </div>
           </section>
+          <?php include 'layout/footer.php'; ?>
+
      </main>
 
-     <?php include 'layout/footer.php'; ?>
```

Comment:
Actually, this is what I meant to be doing. I wanted to keep the layout separated in this way:
header
main (main content of that page)
footer

Reply:
true. not sure why i didnt revert that